src/System.Resources.ResourceManager/tests/System.Resources.ResourceManager.Tests.csproj

```diff
@@ -34,9 +34,17 @@
       <Generator>ResXFileCodeGenerator</Generator>
       <LastGenOutput>TestResx.Designer.cs</LastGenOutput>
     </EmbeddedResource>
+    <EmbeddedResource Include="Resources\TestResx.es.resx"/>
+    <EmbeddedResource Include="Resources\TestResx.es-MX.resx"/>
+    <EmbeddedResource Include="Resources\TestResx.fr-FR.resx"/>
     <SupplementalTestData Include="Resources\TestResx.netstandard17.resources" />
     <SupplementalTestData Include="Resources\bitmap.bmp" />
     <SupplementalTestData Include="Resources\icon.ico" />
+  <!-- TBD Need to copy satellite assemblies into the test directory...  This doesn't work...
+    <SupplementalTestData Include="es\System.Resources.ResourceManager.Tests.resources.dll" />
+    <SupplementalTestData Include="es-MX\System.Resources.ResourceManager.Tests.resources.dll" />
+    <SupplementalTestData Include="fr-FR\System.Resources.ResourceManager.Tests.resources.dll" />
+  -->
```

Comment:
@RussKeldorph @mmitche @sbomer I am having trouble figuring out how to modify the msbuild scripts to get the satellite assemblies copied into the test directory.  Can any of you help or direct me to someone who can?

Reply:
@danmosemsft No, I do not think it helps.  The problem is that build products are not being copied.  I need to figure out the variable name for a build product directory to get stuff copied properly.

Reply:
`$(OutputPath)` seems to work

Reply:
While it seemed to work locally it is not working in CI.  See comment below...